### PR TITLE
Multiple environments: part 2 (antelope)

### PR DESCRIPTION
**Pull request by jovial**

### CODE REVIEW

**Comment by jovial** [comment]
Testing here: https://github.com/stackhpc/stackhpc-kayobe-config/pull/724

**Comment by jovial** [comment]
Failed on SMS lab testing:
```
TASK [kolla-openstack : Ensure extra configuration files exist (strategy: merge_yaml)] **************************************************************************************
Thursday 02 November 2023  18:42:02 +0000 (0:00:00.020)       0:00:31.182 *****                                                                                              
changed: [localhost] => (item={'sources': ['/home/stack/will/antelope/src/kayobe-config/etc/kayobe/kolla/config/glance/policy.yaml'], 'dest': '/home/stack/will/antelope/src/
kayobe-config/etc/kolla/config/glance/policy.yaml', 'params': []})                                                                                                           
changed: [localhost] => (item={'sources': ['/home/stack/will/antelope/src/kayobe-config/etc/kayobe/kolla/config/grafana/provisioning.yaml'], 'dest': '/home/stack/will/antelo
pe/src/kayobe-config/etc/kolla/config/grafana/provisioning.yaml', 'params': []})                                                                                             
An exception occurred during task execution. To see the full traceback, use -vvv. The error was:                   ^                                                        
fatal: [localhost]: FAILED! =>                                                                                                                                               
  msg: |-                                
    Unexpected failure during module execution: while scanning for the next token
    found character '%' that cannot start any token
      in "<unicode string>", line 27, column 15:
                url: {% raw %} '{{ template "__alert_s ...
                      ^
  stdout: ''
  ```
  As not valid yaml. We could be more backwards compatible if we fallback to templating files with a single source.

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -447,6 +447,79 @@
 #kolla_enable_watcher:
 #kolla_enable_zun:
 
+###############################################################################
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml:450]: I suppose this should go into SKC.

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (jovial) [reply etc/kayobe/kolla.yml:450]: Good call, proposed here: https://github.com/stackhpc/stackhpc-kayobe-config/pull/778

**Pull request merged**